tests/integration/content_types/test_406_errors.py

```diff
@@ -22,7 +22,10 @@ def test_406(
     method,
 ):
     """
-    .. py:function:: Test 406 response
+    .. py:function:: Scenario: An API consumer submitting a request with an \
```

Comment:
I think you could put this scenario text in the partials aswell, want to give it a try?